move/squid/sources/squid/transfers.move

```diff
@@ -21,6 +21,7 @@ public struct SuiTransferSwapData has drop {
     swap_type: u8,
```

Comment:
define an enum for SwapType (in a separate PR if you prefer). It can still be stored as `u8` since Sui doesn't allow modifying enums in upgrades, but at least it's passed around internally in a typed manner?